src/nft.vy

```diff
@@ -567,6 +573,23 @@ def safe_mint(owner: address, uri: String[432]):
     self._safe_mint(owner, token_id, b"")
     self._set_token_uri(token_id, uri)
 
+@external
+def mint_and_start_auction(uri: String[432], patron: address):
```

Comment:
Why don't we also have batch_mint function?

Reply:
nack: isnt required for the auction scope / curation mechanism, and would increase complexity. Since we have opted for ERC721 instead of ERC1155 ( for compatibility  / ease of use ), seems okay to keep as it is.

Reply:
It's unrelated to token standards, It's about having option to optimise for gas by batching the NFT mints in a single call. Curators can batch the minting of several NFTs under a single call to conserve gas and have ease of use.

Reply:
tbd